### PR TITLE
Parallelize pytest (#1)

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

File: .github/workflows/create-test-lint-wf-template.yml [hunk 1]
```diff
@@ -20,17 +20,17 @@ env:
...
     strategy:
       matrix:
         TEMPLATE:
-          - "template_skip_all.yml"
+          # - "template_skip_all.yml"
           - "template_skip_github_badges.yml"
           - "template_skip_igenomes.yml"
           - "template_skip_ci.yml"
-          - "template_skip_nf_core_configs.yml"
+          # - "template_skip_nf_core_configs.yml"
```
[the remark this answers is not in the document]
Comment (mashehu) [reply .github/workflows/create-test-lint-wf-template.yml:33]: yes, good idea. don't know how else to handle them.

**Review by mirpedrol — APPROVED** [review]
Awesome!

**Comment by maxulysse** [comment]
so self hosted runners are working well?

**Comment by mashehu** [comment]
yes. took a bit, but now everything is fine. next step: use them in modules (tested them already in the bedtools version bump PR and worked fine except for some api rate limits, which I am now better able to circumvent by having things preinstalled on the aws runner)

**Comment by maxulysse** [comment]
then GPUS?

**Comment by mashehu** [comment]
also on the list

**Pull request merged**